docs/DeveloperGuide.md

```diff
@@ -553,7 +553,7 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar socialbook.jar` command to run the application. Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```

Comment:
I think the last word should be "optimal"? Also do you think it would be good to have a mention of using `java -version` to check the correct version of java?

Reply:
changed to "optimal". 

regarding the 2nd point... I think it is not necessary given that this belong in the DG section and at the start there is a setting up section.